Makefile

```diff
@@ -186,14 +186,18 @@ stylesheets: public/css/index.css
 
 .IGNORE: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
-	lessc $< $@
+	@hash lessc > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		docker run --rm -v $(PWD):/app appleboy/node-less $< $@; \
+	else \
+		lessc $< $@; \
+	fi
```

Comment:
It might be better to just depend on `node` and `npm` and then install `node-less` as a local dependency (with a `package.json`) than to depend on docker.